packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -214,12 +214,22 @@
                             ></dt>
                             <dd
                                 class="oui-tile__description"
-                                data-translate="{{'server_datacenter_' + $ctrl.server.datacenter.split('_')[0]}}"
-                                data-translate-values="{
-                                    number: $ctrl.server.datacenter.split('_')[1]
-                                  }"
-                            ></dd>
+                            >
+                                <span data-ng-bind=":: $ctrl.server.datacenter.toUpperCase().split('_').join('') + ' - ' + ( 'server_datacenter_' + $ctrl.server.datacenter.toUpperCase().split('_')[0] | translate )"
+                                >
+                                </span>
+                            </dd>
                         </dl>
+                        <div class="d-flex align-items-center">
+                            <h2
+                                data-ng-hide=":: !$ctrl.DC_2_ISO[$ctrl.server.datacenter.toUpperCase().split('_')[0]]"
+                                data-oui-tooltip="{{:: 'country_' + $ctrl.DC_2_ISO[$ctrl.server.datacenter.toUpperCase().split('_')[0]].toUpperCase() | translate }}"
+                                data-oui-tooltip-placement="bottom"
+                                class="oui-flag {{ 'oui-flag_' + $ctrl.DC_2_ISO[$ctrl.server.datacenter.toUpperCase().split('_')[0]].toLowerCase() }}"
```

Comment:
**nitpick:** `$ctrl.DC_2_ISO[$ctrl.server.datacenter.toUpperCase().split('_')[0]]` this is repeated multiple times. Can we compute it once and store it in `$ctrl` to avoid re-evaluating multiple times?